### DISCUSSION TITLE
【注意】大家要小心一个人！

### DISCUSSION THREAD

**Discussion by dolokong** [share121/inter-knot#1894] [其他]
因为事情太紧急，没来得及发照片。
事情是这样的：我约朋友在光映广场见面之后，在开车回六分街的路上发现一个黝黑的男人从空中掉了下来，我开车过去想看看请况，谁知道那家伙一下子起来肘击我的车门，我的副驾驶车门都被肘变形了！幸亏跑得快，本来想过去帮忙，谁知道会碰到这种奇葩，这家伙不会被以太侵蚀了吧？有没有懂哥告诉我一下，我想要联系治安官了！
大家出门一定要注意啊！那人穿着24号黄色篮球服！

**Comment by gaojiaobei** [comment]
不好，高级遗骸[坠机]！
为啥会在空洞外出现?

**Comment by uyfh352** [comment]
@ZhuYuan-148 出來一下

**Comment by ZhuYuan-148** [comment]
请这位市民再多详细描述一下！最好带上图片一起
